docs/components/alerts/index.html

```diff
@@ -1,161 +1,114 @@
+            </option>
+          </select>
+        </p>
       </div>
-    </div>
-    {% code 'html' %}
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert">
-          <div class="hxAlert__icon">
-            <hx-icon type="info-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Info</span>
-            Here is helpful information.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Acknowledge</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
-        </div>
-      </div>
-    {% endcode %}
-  </section>
-
-  <section>
-    <h2 id="error">Error</h2>
 
-    <div class="demo">
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert hxAlert--error">
-          <div class="hxAlert__icon">
-            <hx-icon type="exclamation-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Error</span>
-            Change a few things up and try submitting again.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Correct Issue</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
-        </div>
-      </div>
-    </div>
-    {% code 'html' %}
-      <div role="status" class="hxAlerts" aria-live="polite">
-        <div class="hxAlert hxAlert--error">
-          <div class="hxAlert__icon">
-            <hx-icon type="exclamation-circle"></hx-icon>
-          </div>
-          <div class="hxAlert__text">
-            <span class="hxAlert__status">Error</span>
-            Change a few things up and try submitting again.
-          </div>
-          <div class="hxAlert__action">
-            <a href="#">Correct Issue</a>
-          </div>
-          <button class="hxAlert__dismiss"><hx-icon type="times"></hx-icon></button>
+      <div class="hxCol hxSpan-12-xs hxSpan-9-lg hxOrder-1-lg">
+        <h3>Basic Alert</h3>
+        <div class="demo alert-demo">
+          <hx-disclosure :disabled="!isClosable" class="hxBtn" aria-controls="myAlert">
+            Toggle Alert
+          </hx-disclosure>
+          <br /><br />
+          <hx-alert id="myAlert" :closable="isClosable" v-bind:class="cssClasses">
+            <hx-alert-description>
+              <span class="hxAlertStatus">Info</span>
+              Here is helpful information.
+            </hx-alert-description>
+            <hx-alert-action>
+              <a href="#">Acknowledge</a>
+            </hx-alert-action>
+          </hx-alert>
         </div>
+        <template v-if="cssClasses === '' && !isClosable">
+          {% code 'html' %}
+            {% raw %}
+              <hx-alert>
+                <hx-alert-description>
+                  <span class="hxAlertStatus">Info</span>
```

Comment:
This implementation isn't sitting quite right with me. Here, I see both `<hx-*>` prefixed _elements_ as well as an `.hx*` prefixed _CSS class_.  Given that there shouldn't be anything other than text within the status, it would work well as an HTML attribute (e.g. `<hx-alert status="Info">`). If the attribute isn't present (or is blank), we can just hide the status element within the shadow dom.

_proposed_
```html
<hx-alert status="Info">
  <hx-alert-description>...</hx-alert-description>
  <hx-alert-action>...</hx-alert-action>
</hx-alert>
```

Reply:
That makes things cleaner, I've made the change.